dymos/ode_options.py

```diff
@@ -229,7 +229,8 @@ def declare_time(self, targets=None, units=None):
 
         self._check_targets('time', self._time_options['targets'])
 
-    def declare_state(self, name, rate_source, targets=None, shape=None, units=None):
+    def declare_state(self, name, rate_source, targets=None, shape=None,
+                      units=None, solved_segments=False):
```

Comment:
solved_segments should not be set at the ODE level, its a phase-specific option.